lib/ruby_indexer/lib/ruby_indexer/visitor.rb

```diff
@@ -46,6 +46,18 @@ def visit(node)
 
         name = fully_qualify_name(node.target.location.slice)
         add_constant(node, name)
+      when YARP::MultiWriteNode
+        value = node.value
+        values = value.is_a?(YARP::ArrayNode) ? value.elements : Array.new(node.targets.length, value)
```

Comment:
I think you can write this and not have an ArrayNode: `A, B = C`

Reply:
Yes. Added a test for this as well. In that case, `A` is an `UnresolvedAlias` and `B` is just a declared constant for which the value is `nil`.